CHANGES.md

```diff
@@ -15107,7 +15069,7 @@ ndif
 
  * Change names of new functions to the new get1/get0 naming
    convention: After 'get1', the caller owns a reference count
```

Comment:
Backticks around `` `get1` ``?

Reply:
(same for `` `get0` `` and `` `get` `` below)

Reply:
another PR to fix the pipe and quote "markup" is a good idea.  maybe someone will do that.  maybe i will :)  i would like my other doc update PR's to get merged first.